### ISSUE TITLE
Map Instantiation Failed Errors

### ISSUE THREAD

**Issue by michellechandra** [CartoDB/cartodb#10675]
### Context
**User:** michellemho-carto (team account)
**Map URL:** https://team.carto.com/u/michellemho-carto/builder/4cf86e86-a6b4-11e6-ab8e-0ef7f98ade21
**Map Name:** Election 2016

Michelle was using her account to apply different analysis options to her Election 2016 map when all of a sudden, her Builder account crashed. When she tries to access the map from her dashboard, it no longer displays data or the Builder UI. There are several console errors.

Can this map be restored for Michelle? Please reassign if I have assigned to the wrong manager.
 
### Map Dashboard
![screen shot 2016-11-09 at 14 14 41](https://cloud.githubusercontent.com/assets/5423288/20157004/a50db87e-a687-11e6-928e-e5eba6adb24b.png)

### Current Map View
![screen shot 2016-11-09 at 14 14 53](https://cloud.githubusercontent.com/assets/5423288/20157023/b1aa73ec-a687-11e6-8c7c-07573c87e9b8.png)

### Console Errors
![screen shot 2016-11-09 at 14 17 54](https://cloud.githubusercontent.com/assets/5423288/20157032/b84d1416-a687-11e6-861e-4ce951110731.png)

### .carto file
Not available at this time.

cc  @CartoDB/support 
cc @michellemho

**Comment by ernesmb** [comment]
looks like a 'ghost node' error?

**Comment by xavijam** [comment]
Yep, looks like so, checking it.

**Comment by xavijam** [comment]
Map fixed, also @gfiorav is checking a possible "help" for this problem. 💋

**Comment by xavijam** [comment]
Hey @michellechandra, what did you do in order to show that error? Did you drag out something? add a new layer... dod you remember it?

**Comment by michellemho** [comment]
@xavijam I was working on this map. I dragged out an analysis as a new layer when I got this error. I just learned that we could do this, so I got excited and dragged out several analyses to experiment. I got a warning that I could not have more than 8 layers. I deleted some layers, and continued to drag until I got the error and my map crashed. Hope this helps you figure out the error... And thanks for fixing the map!
